Gemfile

```diff
@@ -7,5 +7,6 @@ group :local_development do
   gem 'guard-rspec'
   gem 'appraisal'
   gem 'rake'
+  gem 'sqlite3'
```

Comment:
I added that because it's a requirement when working on the gem

without it:
```
>$ bundle
...
>$ bundle exec rake rspec
```

![image](https://user-images.githubusercontent.com/12391171/122456978-25ae3400-cf7c-11eb-996d-a18ad1f74c97.png)

Reply:
Strange , i never needed to add it. You can use appraisal to run a specific version.